src/translations/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+project(qps)
```

Comment:
What is this CMakeLists files good for?

Reply:
Without it it didn't include the translations in qps.desktop, only the template.
Similar files are present  in other foo/translations directories.

Reply:
If I'm not mistaken, this was for building when the translations were built in lxqt-l10n. It definitely isn't needed for the `lxqt_translate_desktop`.

Reply:
I'll retest on debian 32bit, I know little about cmake.
So those could be removed from all translation folders?

Reply:
retested, don't know what happened yesterday, it's not needed. Doesn't make sense also.
Removed the file.